Traceback (most recent call last):
  File "/gdata/huangzp/hr_regan/main.py", line 41, in <module>
    main(config)
  File "/gdata/huangzp/hr_regan/main.py", line 29, in main
    trainer = Trainer(config, a_data_loader, b_data_loader)
  File "/gdata/huangzp/hr_regan/trainer.py", line 56, in __init__
    self.build_model()
  File "/gdata/huangzp/hr_regan/trainer.py", line 105, in build_model
    conv_dims[-1]/2, b_channel, deconv_dims, self.num_gpu)
  File "/gdata/huangzp/hr_regan/models.py", line 157, in __init__
    self.layers.append(nn.ConvTranspose2d(prev_dim, out_dim, 4, 2, 1, bias=False))
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/conv.py", line 595, in __init__
    True, output_padding, groups, bias)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/conv.py", line 30, in __init__
    in_channels, out_channels // groups, *kernel_size))
TypeError: torch.FloatTensor constructor received an invalid combination of arguments - got (float, int, int, int), but expected one of:
 * no arguments
 * (int ...)
      didn't match because some of the arguments have invalid types: (!float!, !int!, !int!, !int!)
 * (torch.FloatTensor viewed_tensor)
 * (torch.Size size)
 * (torch.FloatStorage data)
 * (Sequence data)

Unexpected end of /proc/mounts line `overlay / overlay rw,relatime,lowerdir=/home/docker-root/overlay2/l/VYEKTA4MZ3NQFF7OCRT6QDGWLU:/home/docker-root/overlay2/l/AZKAXIRQMH7POJIQEDKXQI4FIL:/home/docker-root/overlay2/l/IEHICCHRNWVNEFPDII5V4UOVNJ:/home/docker-root/overlay2/l/M2UN22Y23ISVSR35AGR5F5QUWO:/home/docker-root/overlay2/l/5YEGEGXKBZ23DXJGWNEIPLBSFT:/home/docker-root/overlay2/l/A62L5NEWBDNRKNZ4HZ7L2F2KRL:/home/docker-root/overlay2/l/PMEIESVLUFSJ6KH5NU7B3QG3JB:/home/docker-root/overlay2/l/2Z7A4XNYQPC7MVMVOM4RQMHNVW:/home/docker-root/overla'
